### ISSUE TITLE
.gitkeep in plugins causes ENOTDIR

### ISSUE THREAD

**Issue by articice** [strapi/strapi#3803]
**Describe the bug**
Creating a .gitkeep file in plugins directory causes an error preventing Strapi from starting.
```
error Error: ENOTDIR invalid cwd /home/user/strapi-test/plugins/.gitkeep/middlewares
    at Glob._readdirError (/home/user/Socure/dev-hub/node_modules/glob/glob.js:592:21)
    at /home/user/Socure/dev-hub/node_modules/glob/glob.js:553:12
    at FSReqCallback.oncomplete (fs.js:153:23)
error Command failed with exit code 1.
```

**Steps to reproduce the behavior**
1. touch plugins/.gitkeep
2. strapi develop 
3. See error

**Expected behavior**
Plugins directory should be manageable via Git.

**System**
- Node.js version: 12.8.0
- NPM version:6.10.2
- Strapi version: 3.0.0-beta.15
- Database: sqlite
- Operating system: Linux

**Comment by alexandrebodin** [comment]
Hi @articice, 

Can you give me the reason you need to have a .gitkeep file in this folder ?

**Comment by articice** [comment]
Hi @alexandrebodin, I developed a plug-in and put it into a separate repository.
Instead of putting it into project dependencies, I created a post-install script to check out the repository into `plugins/`. I put a .gitkeep to already have the directory empty in the main repository and then populate it with plugins. 
With this bug the workaround has to be 'mkdir -p plugins && git clone ... plugins/my-plugin'.
